app/web/src/helpers/hooks/useIdeState.ts

```diff
@@ -6,6 +6,14 @@ import type {
   ArtifactTypes,
 } from 'src/helpers/cadPackages/common'
 import { CadhubParams } from 'src/components/Customizer/customizerConverter'
+import openScadGuide from 'src/helpers/cadPackages/openScad/userGuide.md'
+import openScadInitialCode from 'src/helpers/cadPackages/openScad/initialCode.SCAD'
+import cadQueryGuide from 'src/helpers/cadPackages/cadQuery/userGuide.md'
+import cadQueryInitialCode from 'src/helpers/cadPackages/cadQuery/initialCode.py'
+import jsCadGuide from 'src/helpers/cadPackages/jsCad/userGuide.md'
+import jsCadInitialCode from 'src/helpers/cadPackages/jsCad/initialCode.jscad'
```

Comment:
These work but cause TS errors in VS Code. Unsure how to fix but will investigate, I have a hunch that `.d.ts` files might help this?

Reply:
Got it fixed with `globals.d.ts` file